### PR TITLE
Reexport winit::platform::android::activity::* in bevy_winit

### LINKED ISSUE #10630: Re-export winit::platform::android::activity::* in bevy_winit
## What problem does this solve or what need does it fill?

I want to set a fullscreen flag in my android game, but since the bevy_winit didn't re-export it, I can't set it.

code like this
```
    #[cfg(target_os = "android")]
    {
        use bevy::winit::platform::android::activity::WindowManagerFlags;
        let android_app = bevy::winit::ANDROID_APP
            .get()
            .expect("Bevy must be set up with the #[bevy_main] macro on Android");
        android_app.set_window_flags(
            WindowManagerFlags::FULLSCREEN,
            WindowManagerFlags::FORCE_NOT_FULLSCREEN,
        );
    }
```
will simply give a compile error.

## What solution would you like?

Just add a pub use in bevy_winit crate I think.

## What alternative(s) have you considered?

no for now

## Additional context

At first I want to simply re-export winit::platform::android::activity::WindowManagerFlags, but I think maybe others need more than this.

### PR DESCRIPTION
**Pull request by hecksmosis**
# Objective

- Fixes #10630

## Solution

- Reexport winit::platform::android::activity::* in bevy_init

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

**Comment by JMS55** [comment]
Instead of a top-level export, maybe we want to scope it under a module? I don't have a preference either way I don't think.

**Comment by matiqo15** [comment]
I agree with @JMS55, though it's fine either way.

File: crates/bevy_winit/src/lib.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ use bevy_window::{
 use bevy_window::{PrimaryWindow, RawHandleWrapper};
 
 #[cfg(target_os = "android")]
-pub use winit::platform::android::activity::AndroidApp;
+pub use winit::platform::android::activity::*;
```
Comment (mockersf) [inline crates/bevy_winit/src/lib.rs:50]: maybe
```suggestion
pub use winit::platform::android::activity as android_activity;
```

**Review by matiqo15 — APPROVED** [review]

**Review by mockersf — CHANGES_REQUESTED** [review]
you'll also have to update the places where the `AndroidApp` was used

**Pull request merged**